src/components/subscriptions/styles/_SubscriptionManagementPage.scss

```diff
@@ -9,13 +9,45 @@
     }
   }
 
-  .subscription-card,
   .support-card {
     .card-title {
       @extend .h4;
     }
   }
 
+  .subscription-card {
+    margin-bottom: 1.875rem;
+
+    .card-body {
+      background: $light-200;
+    }
+
+    .badge {
+      padding: 0.3rem 0.5rem;
```

Comment:
Curious why we need to override the default styles on `.badge`? I think we should just use whatever badge styles Paragon gives us by default.

Reply:
The default badge styles doesn't come with any padding for the badges and it looks quite squished.

![Screen Shot 2021-11-02 at 11 23 37 AM](https://user-images.githu
![Screen Shot 2021-11-02 at 11 24 54 AM](https://user-images.githubusercontent.com/17792243/139877176-ca75aa12-ebb3-4df2-a72b-092095a75c7b.png)
busercontent.com/17792243/139877096-7f962720-7986-40d3-9c1a-77906af19e56.png)
.

Reply:
fixed, issue was with the style import order.